docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/src/Plugin/Validation/Constraint/SectionParentTermConstraintValidator.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Drupal\cgov_site_section\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Validates the Section ParentTerm Constraint.
+ *
+ * @package Drupal\cgov_core\Plugin\Validation\Constraint
+ */
+class SectionParentTermConstraintValidator extends ConstraintValidator implements ContainerInjectionInterface {
```

Comment:
The same as with `SectionParentTermConstraint`, a comment describing what's _intended_ is even more important here since this is where it's supposed to happen.

Reply:
Hi Blair, Can you help me with what exactly I need to comment. I looked around and saw the following constraints in the same module and followed same commenting/description style - docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/src/Plugin/Validation/Constraint/UniqueCgovUrlConstraintValidator.php and UniqueCgovUrlConstraint.php.

Reply:
Hi Blair, I added extra comments, please check if all looks satisfactory!